dune-project

```diff
@@ -16,7 +15,6 @@
   (astring (>= 0.8.5))
   (sexplib (>= v0.14.0))
   (cmdliner (>= 1.1.0))
-  (dune-private-libs (and (>= 2.8.0) (< 3.0)))
   (dune (< 3.0))
```

Comment:
This should target a newer version of dune and add the `stdune` library dependency to match.